package.json

```diff
@@ -33,7 +33,10 @@
   "dependencies": {
     "husky": "^7.0.4",
     "lint-staged": "^12.0.2",
-    "prettier": "^2.4.1"
+    "prettier": "^2.4.1",
+    "qrcode.react": "^3.1.0",
+    "react": "^17.0.2",
```

Comment:
Can you please install the dependencies within the extension folder?
To do this you can do:
```
yarn workspace @gemwallet/extension add react
```
Note that react is already installed within the extension package here: `packages/extension/package.json`

Reply:
I've refactor the dependencies and clean workspace settings.